natsjobs/driver.go

```diff
@@ -195,29 +194,20 @@ func FromPipeline(tracer *sdktrace.TracerProvider, pipe jobs.Pipeline, log *zap.
 		return nil, errors.E(op, err)
 	}
 
-	js, err := conn.JetStream()
+	js, err := jetstream.New(conn)
 	if err != nil {
 		return nil, errors.E(op, err)
 	}
 
-	var si *nats.StreamInfo
-	si, err = js.StreamInfo(pipe.String(pipeStream, "default-stream"))
-	if err != nil {
-		if stderr.Is(err, nats.ErrStreamNotFound) {
-			si, err = js.AddStream(&nats.StreamConfig{
-				Name:     pipe.String(pipeStream, "default-stream"),
-				Subjects: []string{pipe.String(pipeSubject, "default")},
-			})
-			if err != nil {
-				return nil, errors.E(op, err)
-			}
-		} else {
-			return nil, errors.E(op, err)
-		}
-	}
+	defStream := pipe.String(pipeStream, "default-streamID")
+	defSubject := pipe.String(pipeSubject, "default.*")
```

Comment:
```suggestion
	defSubject := pipe.String(pipeSubject, "default")
```

Reply:
Without `.*` the error from nats api is `subjects overlap with an existing stream`.

Rest suggestion are applied

Reply:
Hm, I guess we should remove the stream at the test Cleanup `t.Cleanup(func(){}))` routine to satisfy the new API.

Reply:
Looks like it wasn't needed! Tests green 🟢